rpcs3/Emu/Cell/lv2/sys_ppu_thread.cpp

```diff
@@ -430,11 +430,17 @@ error_code sys_ppu_thread_rename(u32 thread_id, vm::cptr<char> name)
 	}
 
 	constexpr u32 max_size = 27; // max size including null terminator
-	const auto pname = name.get_ptr();
+
+	std::string_view pname = name.get_ptr();
```

Comment:
Imagine a case where threadname is found at the last 27 bytes of a page without null terminator, with unmapped memory after.
The previous solution would read 27 bytes and stopped correctly, this one access violates at determination the length of string.

Reply:
Fine fine, should be ok now